src/components/HomePage.tsx

```diff
@@ -4,18 +4,9 @@ import { useApiClient } from "../api";
 import Question from "./Question";
```

Comment:
Let's remove the imports that are not used anymore...... so on this page basically all of them ;)

Reply:
Done